### ISSUE TITLE
Failed building JavaScript bundle. Unable to resolve "module" from "node_modules/esm/esm.js"

### ISSUE THREAD

**Issue by Svarto** [amitport/blob#1]
I get this error when Blob 0.1.0 which has esm 3.2.25 as requirement.

I am using it through React Native Expo SDK 39. When building the javascript package bundle to run it on the iOS emulator it crashes with this error.

**Comment by amitport** [comment]
missing the error :)

**Comment by amitport** [comment]
Seems to be related to react native expo itslef (node_modules/esm/esm.js) exists, I'm not sure why it's not loaded correctly in that framework (and I don't experience with it)

**Comment by Svarto** [comment]
@amitport thanks a lot! Actually it is nhost-js-sdk when they updated to 2.2.4 they started using blob.

https://github.com/nhost/nhost-js-sdk/blob/154795abc4fdd305220040c434f3198c1180e8b0/src/Storage.ts#L90

But somehow it doesn't work with React Native - do you have any experience with incompatibility with React Native? As you can see it should be imported and called correctly.

**Comment by amitport** [comment]
No, sorry, I really have no experience with react native. I saw you published in https://github.com/standard-things/esm/issues/890, maybe they could help.
